boards/STM32F103ZET6-F103ZE-Board.md

```diff
@@ -308,40 +309,45 @@ title: "STM32F103ZET6 - F103ZE board"
 
 ## TFT
 
-| Pin   | Connected to |
-| ----- | ------------ |
-|       | NRST         |
-|       | Ground plane |
-|       | PD9          |
-|       | PD10         |
-|       | PE15         |
-|       | PD8          |
-|       | PE13         |
-|       | PE14         |
-|       | PE11         |
-|       | PE12         |
-|       | PE9          |
-|       | PE10         |
-|       | PE7          |
-|       | PE8          |
-|       | PD0          |
-|       | PD1          |
-|       | PD14         |
-|       | PD15         |
-|       | PD5          |
-|       | PD4          |
-|       | PG12         |
-|       | PG0          |
-|       | PB2          |
-|       | PB1          |
-|       | PF8          |
-|       | PF9          |
-|       | N.C.         |
-|       | N.C.         |
-|       | VBAT         |
-|       | Ground plane |
-|       | +3.3V rail   |
-|       | Ground plane |
+| Pin   | TFT  | Connected to | Function       |
+| ----- | ---- | ------------ | -------------- |
+|   2   | RST  | NRST         |                |
+|   1   | GND  | Ground plane |                |
+|   4   | DB14 | PD9          | FSMC D14       |
+|   3   | DB15 | PD10         | FSMC D15       |
+|   6   | DB12 | PE15         | FSMC D12       |
+|   5   | DB13 | PD8          | FSMC D13       |
+|   8   | DB10 | PE13         | FSMC D10       |
+|   7   | DB11 | PE14         | FSMC D11       |
+|  10   | DB08 | PE11         | FSMC D8        |
+|   9   | DB09 | PE12         | FSMC D9        |
+|  12   | DB06 | PE9          | FSMC D6        |
+|  11   | DB07 | PE10         | FSMC D7        |
+|  14   | DB04 | PE7          | FSMC D4        |
+|  13   | DB05 | PE8          | FSMC D5        |
+|  16   | DB02 | PD0          | FSMC D2        |
+|  15   | DB03 | PD1          | FSMC D3        |
+|  18   | DB00 | PD14         | FSMC D0        |
+|  17   | DB01 | PD15         | FSMC D1        |
+|  20   | WR   | PD5          | FSMC NWE       |
+|  19   | RD   | PD4          | FSMC NOE       |
+|  22   | CS_M | PG12         | FSMC NE4       |
+|  21   | RS   | PG0          | FSMC A10       |
+|  24   | SCS  | PB2          | Touch CS       |
+|  23   | SCK  | PB1          | Touch CLK      |
+|  26   | SO   | PF8          | Touch (MI)SO   |
+|  25   | SI   | PF9          | Touch (MO)SI   |
+|  28   | BLK  | Q1 collector | Note 1         |
```

Comment:
I see you had some trouble with creating the links, you can make links like I said earlier: `[Note 1](#note-1)`. The text between the `[ ]` becomes the text you will see and the text between `( )` is the actual URL. In this case it refers the the headers you created below.

Reply:
You can find an example of how this works on this page: https://github.com/STM32-base/STM32-base.github.io/blob/master/cheatsheets/device-name-mapping.md

Reply:
I looked some further into the backlight circuitry, and did some test with two displays: one with backlight activated low, one with backlight activated high. (It's not really high and low - the backlight driver on the TFT pcbs are simply transistors so there is a gradual change) With R19 uninstalled (as F103ZE board is delivered) the BL-pin is low. The display on the left in the picture works fine. This is a display sellers often try to sell with STM boards, like here: https://www.aliexpress.com/item/32689262341.html
The backlight on the other display stays unlit. 
Shorting the two R19 pads with a blob of solder:
  Makes the BL-pin go to about 2.66V
  Makes the "sold for STM" display go VERY dim, but not quite dark. 
  The other display, often advertised as for Arduino like here: 
  https://www.aliexpress.com/item/4000213883774.html , lights up normally. 
  (This display has a different pin-out and I therefore use it with an adapter)
![IMG_20191128_212150258](https://user-images.githubusercontent.com/37983318/69833612-4fd2c080-1235-11ea-9237-2cf1bad12a27.jpg)

Should we say something like "If the backlight of your display does not light up, you can try to short the two pads of R19 together."  ?

Reply:
By the way, are they REALLY claiming that the blue board has a 3.2 FEET display ?

Reply:
As far as I know, a transistor requires a base resistor. So the advice should be to install a 1k ohm resistor. Do you happen to know the size of the pad? Something like 0602?

And I guess that there's just a typo on the blue board.

Reply:
I tried different base resistors. 1K is too much, 100 ohms works, at least with this display. With 100 ohms the base current is about 0.4mA. 
Sure it's a typo, byt kind of funny, feet instead of inches. Could have been nice with displays that size!

Reply:
Yes, it's an 0603 footprint. Slightly too large to comfortably short out with solder.

Reply:
I see on the schematics for the F407Z-board (Issue #14) there is also an uninstalled resistor in the same place on the schematics (The board layout is slightly different), but there is ALSO a 1K5 resistor from the base of the transistor to a pin (PB15). I will try modifying THIS (blue F103Z-) board in the same way.  Testing...

Reply:
So the things left to do from this conversation are:

- Adding that the resistor has to be a 0603 SMD one
- Recommend a suitable resistor
- Adding whatever your test results are

Thanks for your testing by the way, this kind of information really adds value to the website.

Reply:
1. I STILL haven't got the Note/link thing under control, but gave it up as the Note and the Note text are just a few lines apart.
2. Note 1 has grown rather large, but I think all the relevant information is there.

Reply:
Ok, that's fine.